microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -5,147 +5,223 @@ author: MSEdgeTeam
+All aspects of your extension should accurately describe the functions, features and any important limitations of your extension, including required or supported input devices.  The value proposition of your extension must be clear during the first run experience.  Your extension may not use a name or icon similar to that of other extensions, must not reference other browsers and must not claim to represent a company, government body, or other entity if you don't have permission to make that representation.
 
-#### 1.1.3 Functionality
+If screenshots are provided, they must:
+- Be clear and informative.
+- Render properly; for example, must not be stretched or blurry.
 
-Your extension must be fully functional.
+All functionality of extensions should be clearly disclosed to the user. Any extensions that attempt to deceive or mislead users will be removed from the store.
 
-#### 1.1.4 Search and Discovery
+
+###### 1.1.3 Functionality
+
+Your extension must be fully functional as in there should not be any blockers including, but not limited to, broken URLs, blocked links and pages.
+
+
+###### 1.1.4 Search and discovery
 
 Search terms may not exceed seven unique terms and should be relevant to your extension.
 
-#### 1.1.5 Provide appropriate details
 
-You must provide distinct and informative details about your extension and the functionality in listing (metadata) for your extension.  Your extension must provide a valuable and quality user experience.  Your extension must also have an active presence in Microsoft Edge Add-ons.
+###### 1.1.5 Provide appropriate details
+
+There should be distinct and informative details about your extension and the functionality in listing (metadata) for your extension.
+
+
+###### 1.1.6 Stability and performance
 
-#### 1.1.6 Stability and Performance
+Developers must ensure that their extensions are stable, do not cause the browser to freeze or crash.
 
-Your extension must not negatively impact the performance or stability of Microsoft Edge.
+In general, the extension cannot interfere with the normal functioning of the browser or operating system.
 
-#### 1.1.7 Obfuscation
+
+###### 1.1.7 Obfuscation
 
 Extensions with obfuscated code aren't allowed.  This includes code within your extension package as well as any external code or resource fetched from the web.  You may be asked to refactor parts of your code if it is not reviewable.
 
-#### 1.1.8 Altering Browser Settings
+
+###### 1.1.8 Altering browser settings
 
 Your extension must not, without appropriate user consent, alter, or appear to alter, browser functionality or settings including, but not limited to: the address bar search provider and suggestions, the start or home page, the new tab page, and adding or removing favorite.
```

Comment:
"favorites"

Reply:
the new tab page, and adding or removing favorites.